src/data/fintype/basic.lean

```diff
@@ -1412,6 +1445,26 @@ begin
   simp
 end
 
+@[simp]
+lemma fintype.card_subtype_compl [fintype α]
+  (p : α → Prop) [fintype {x // p x}] [fintype {x // ¬ p x}] :
+  fintype.card {x // ¬ p x} = fintype.card α - fintype.card {x // p x} :=
+begin
+  classical,
+  rw [fintype.card_of_subtype (set.to_finset pᶜ), set.to_finset_compl p, finset.card_compl,
+      fintype.card_of_subtype (set.to_finset p)];
+  intros; simp; refl
```

Comment:
Are we missing a simp lemma?

Reply:
This is code that had previously been through review, but I went ahead and fixed the non-terminal `simp`.

What's going on is that this proof is that `p` is not a `set` but the rewrites are using it as if it were one. In a quick test, `set_of p` doesn't fix it.